ci/test_python.sh

```diff
@@ -117,7 +117,7 @@ if [[ "${RAPIDS_CUDA_VERSION}" == "11.8.0" ]]; then
       --channel "${PYTHON_CHANNEL}" \
       --channel pytorch \
       --channel pytorch-nightly \
-      --channel dglteam/label/cu117 \
+      --channel dglteam/label/cu118 \
```

Comment:
Ahh, just saw that that they have cu118 builds here: https://anaconda.org/dglteam/dgl 


Surprisingly it is missing on  getting started page: https://www.dgl.ai/pages/start.html